### DISCUSSION TITLE
fullName, 비밀번호 validation

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#176] [Ideas]
@hyunjin0910 😍
현진님 혹시 회원가입 fullName, 비밀번호 폼 검증 만드셨거나 생각 중인 검증이 있으신가요??
오늘 프로필 페이지 fullName 변경, 비밀번호 변경 만들려고 하는데 검증 맞춰야 할 것 같아서요!!
아직 안 하셨다면 같이 검증 조건 맞추고, 제가 만들겠습니다.

**Comment by YuHyun-P** [comment]
- fullName: 한글/영문/숫자 2-8자
- 비밀번호: 영문/숫자 6-20자
로 생각하고 있습니당

**Reply by hyunjin0910** [reply]
비밀번호는 6-12자 조건으로 짜고 있는데 이름 조건은 아직 생각 안해봤어요! 저렇게 맞추시죠~

**Reply by YuHyun-P** [reply]
그러면
- fullName: 2-8자 한/영/숫자
- 비밀번호: 6-12 영/숫자 조합

맞죠?!
오늘 내로 만들어서 올릴게욤
